### ISSUE TITLE
[Enhancement] Allow editing bridge configurations

### ISSUE THREAD

**Issue by t0bst4r** [t0bst4r/home-assistant-matter-hub#25]

**Comment by kramttocs** [comment]
By changing the config so that devices are added or removed in the hub, how will that impact the assistant device?
Meaning will they automatically be added or removed from it as well?

**Comment by t0bst4r** [comment]
Yes and no, I tested that with Alexa by using a label to include entities. By adding or removing the label to another device and restarting the addon you can achieve the edit functionality. 

When adding an entity (and restarting the addon) Alexa automatically finds the new devices after some minutes. You can force it manually by asking to search for devices. But that is not necessary. 
When removing a device it does not disappear from Alexa. Instead it gets "unreachable" and you need to remove it by hand.
